sdk/servicebus/azure-servicebus/README.md

```diff
@@ -163,11 +168,13 @@ with ServiceBusClient.from_connection_string(connstr) as client:
     # idle_timeout specifies how long the receiver should wait with no incoming messages before stopping receipt.  
     # Default is None; to receive forever.
     with client.get_queue_receiver(queue_name, idle_timeout=30) as receiver:
-        for msg in receiver:
+        for msg in receiver:  # ServiceBusReceiver instance is a generator
             print(str(msg))
 ```
 
-> **NOTE:** `ServiceBusReceiver.receive()` provides another mechanism to receive if it is desired to receive a single or constrained batch of messages through a single method call, as opposed to receiving perpetually.  For instance:
+#### Receive messages from a queue through `ServiceBusReceiver.receive()`
+
+> **NOTE:** `ServiceBusReceiver.receive()` always returns an array of messages.
```

Comment:
I'm a little hesitant to remove the "as opposed to receiving perpetually" line but I do think the terseness upfront is worthwhile

Reply:
I can put the note back under this section, will tweak.